KPI/views/firefox_desktop_usage_2021.view.lkml

```diff
@@ -136,6 +136,7 @@ derived_table: {
   }
 
   measure: new_profiles_cumulative_2021 {
+    hidden: yes
```

Comment:
I ended up hiding a bunch of fields we specifically need for the topline dashboard numbers. I don't expect people would want to query these in the explore.